pkgs/applications/networking/cloudflared/default.nix

```diff
@@ -13,7 +13,13 @@ buildGoModule rec {
 
   vendorSha256 = null;
 
-  doCheck = false;
+  postPatch = ''
```

Comment:
Please add some comments why the tests fail.

Reply:
Done